nina_xmpp/__main__.py

```diff
@@ -16,7 +17,10 @@ def main():
     config = yaml.safe_load(args.config_file)
     args.config_file.close()
 
-    main = NinaXMPP(config)
+    setup_config = configparser.ConfigParser()
+    setup_config.read('setup.cfg')
```

Comment:
This assumes that I'm running the command from a git checkout, right? I'd like it to be compatible with installation via `pip`, which probably requires some more elaborate APIs or some configuation in `setup.cfg` to copy itself into the installation directory.

Reply:
Oh god, you're right of course. I thought this is clever and was assuming that file be there.. As I have no good idea I removed it from this PR for now.
I could only think of introducing something like a constant, but that would need to be manually bumped when you release/tag which is a downside. The upside is that even -dev suffixes or so would be supported, which indicate that a develop version is running, but I guess there is not much of a need for that :)